src/js/features/progress.js

```diff
@@ -11,7 +11,20 @@ import {secondsToTimeCode} from '../utils/time';
  *
  * This feature creates a progress bar with a slider in the control bar, and updates it based on native events.
  */
-
+	function setCurrentRailMain(t,fakeTime){ 
+		if (t.media.currentTime !== undefined && t.media.duration) {
+		var nTime = (typeof fakeTime === 'undefined') ? t.media.currentTime : fakeTime; 
+		
+			// update bar and handle
+			if (t.total && t.handle) {
+				var newWidth = Math.round(t.total.width() * nTime / t.media.duration),
+				    handlePos = newWidth - Math.round(t.handle.outerWidth(true) / 2); 
+				newWidth = fakeTime / t.media.duration * 100;
```

Comment:
It should be `nTime` instead of `fakeTime`; otherwise, the rail never gets updated